src/index.html

```diff
@@ -6,9 +6,67 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
```

Comment:
Most probably you can remove `<div class="container" ...` and move it's styles to the header

That will make your life easier when styling the header

Reply:
i tried to fix it but it didn't work this way

Reply:
In addition to repositioning the classes and removing the redundant tag, you also need to update the `css` styles to make everything work properly.

Reply:
I've fixed it now, i had to delete width in the header to make it works, thanks.